crates/aranya-client-capi/src/api/defs.rs

```diff
@@ -1368,6 +1625,46 @@ pub unsafe fn query_device_keybundle(
     Ok(KeyBundle::from_underlying(keys))
 }
 
+/// Query device label assignments.
+///
+/// @param client the Aranya Client [`Client`].
+/// @param team the team's ID [`TeamId`].
+/// @param device the device's ID [`DeviceId`].
+/// @param labels returns a list of labels assigned to the device [`LabelId`].
+/// @param labels_len returns the length of the labels list [`LabelId`].
+///
+/// @relates AranyaClient.
```

Comment:
nit: I think even though "returns" is part of the output params, it is still a bit unclear on first glance which params are output.

Reply:
How could we make that more clear?

Reply:
Maybe some spacing between input/output params, with a label?

```suggestion
/// Query device label assignments.
///
/// @param client the Aranya Client [`Client`].
/// @param team the team's ID [`TeamId`].
/// @param device the device's ID [`DeviceId`].
///
/// Output params:
/// @param labels returns a list of labels assigned to the device [`LabelId`].
/// @param labels_len returns the length of the labels list [`LabelId`].
///
/// @relates AranyaClient.
```

Reply:
Doxygen has an in/out modifier, e.g. `@param[out]` Not sure how it actually looks rendered.

Reply:
I'd rather use the attributes which assign semantic meaning. Especially since `labels_len` is both, so `@param[in,out]`. But it can be done on a cleanup PR to update existing as well.